interface/main/finder/dynamic_finder.php

```diff
@@ -17,7 +17,7 @@
 require_once("$srcdir/headers.inc.php");
 
 // Including Bootstrap library.
-call_required_libraries($bootstrap=true,$fancybox=false,$knockout=false,$datepicker=false);
+  call_required_libraries("jquery-min-3-1-1","bootstrap");
```

Comment:
I see a jquery in the older version and it is still there in this version. This must be here for the new bootstrap changes.